frontend-thea-aa/src/routes/+page.svelte

```diff
@@ -4,42 +4,176 @@
 	import ContaminatedSites from '$lib/components/ContaminatedSites.svelte'
 	import Map from '$lib/components/Map.svelte'
 	import { Button } from '$lib/components/ui/button'
-
+	import { Building, Building2, Search } from 'lucide-svelte';
+	import { Cloudy } from 'lucide-svelte';
+	import { Wind } from 'lucide-svelte';
+	import { ListFilter } from 'lucide-svelte';
+	import AirQualityRanking from '$lib/components/AirQualityRanking.svelte'
+   
 	let { data } = $props()
+	let currentView = $state('');
+	let height = $state(0);
+	let mapInstance: any;
+	let apiKey = "672fc9cf41928442178811nqtede4fc"
```

Comment:
no api key in the code, put it in the env file, this doesn't matter that much in this case cause we will get a new env variable now